libandroid-navigation/src/main/java/org/maplibre/navigation/android/navigation/v5/offroute/OffRouteListener.kt

```diff
@@ -0,0 +1,7 @@
+package org.maplibre.navigation.android.navigation.v5.offroute
+
+import android.location.Location
+
+interface OffRouteListener {
```

Comment:
Consider converting single method interfaces to `fun interface` to use SAM conversion

Reply:
Changed ✅

Reply:
Thank you for this change 👍
In the interest of keeping the API consistent, it would probably make sense to convert all such interfaces, not only this one ;)

Reply:
Good point! For `open` I viewed all files again. But for this, I missed that 🤦

Reply:
Now it's hopefully done for all single-function interfaces :-)